include/modules/clustering/hierarchical_clustering_method.hpp

```diff
@@ -37,10 +37,13 @@ int junction_distance(Junction const & lhs, Junction const & rhs);
 /*! \brief Cluster junctions by an hierarchical clustering method.
  *         The returned clusters and the junctions in each returned cluster are sorted.
  *
- * \param[in] junctions - a vector of junctions (needs to be sorted)
+ * \param[in] junctions         - a vector of junctions (needs to be sorted)
  * \param[in] clustering_cutoff - distance cutoff for clustering
+ * \param[in] verbose           - verbose option
  *
  * \details For the algorithms we use the library hclust.
  * \see https://lionel.kr.hs-niederrhein.de/~dalitz/data/hclust/ (last access 01.06.2021).
  */
-std::vector<Cluster> hierarchical_clustering_method(std::vector<Junction> const & junctions, double clustering_cutoff);
+std::vector<Cluster> hierarchical_clustering_method(std::vector<Junction> const & junctions,
+                                                    double clustering_cutoff,
+                                                    bool const verbose);
```

Comment:
I would remove `const` here. Since `bool` (also the `double` above) is passed by copying the value, you will not modify the caller's object anyways.
```suggestion
                                                    bool verbose);
```

Reply:
I slightly disagree. Putting `const` everywhere you can, never hurts (assuming you don't want to change the value of the variable ;-) ).
`const` is just a helper to avoid programming mistakes.
(On the other hand, I am also to lazy to put `const` everywhere)

Reply:
Yeah, that's true, it never hurts. 
I'm just a friend of consistency and if the style in the line above is different, I get alerted :)

Reply:
I see, you truly life by the quote:  "Always code as if the guy who ends up maintaining your code will be a violent psychopath who knows where you live. Code for readability."! :1st_place_medal: